.jenkins/job_beam_PostCommit_Java_RunnableOnService_Dataflow.groovy

```diff
@@ -26,7 +26,7 @@ mavenJob('beam_PostCommit_Java_RunnableOnService_Dataflow') {
   previousNames('beam_PostCommit_RunnableOnService_GoogleCloudDataflow')
 
   // Set common parameters.
-  common_job_properties.setTopLevelMainJobProperties(delegate)
+  common_job_properties.setTopLevelMainJobProperties(delegate, defaultTimeout:120)
```

Comment:
The calling syntax is right, but the receiver would have to accept them as a `Map`, which doesn't seem worth it. Just passing all the args instead.